src/main/java/com/faforever/client/teammatchmaking/InvitePlayerController.java

```diff
@@ -83,28 +112,8 @@ public Pane getRoot() {
     return root;
   }
 
-  public void onInviteButtonClicked(ActionEvent event) {
-    invite();
-  }
-
-  private void invite() {
-    playersListView.getSelectionModel().getSelectedItems().forEach(player -> {
-      teamMatchmakingService.invitePlayer(player);
-      invitedPlayersListView.getItems().add(player);
-    });
-    playerTextField.setText("");
-    JavaFxUtil.runLater(() -> playersListView.getSelectionModel().selectFirst());
-  }
-
-  public void onKeyPressed(KeyEvent keyEvent) {
```

Comment:
Please re-add inviting by pressing enter and keyboard navigation of the invite list. It helps a lot when you want to invite multiple people.

Reply:
I find that super not intuitive this is exactly what I want to get rid of. I want to confirm my search so I press enter. But instead of searching it invites the player on top of the list. U can not imagine how many ppl I  invited by accident.

Reply:
Yes it is definitely slower. But who expects such behavior of a search dialog.

Reply:
And i did not think that was a time critical thing here... Where it is important if I need 10 or 13 seconds to invite my friends

Reply:
i think I don't understand your workflow. You said you wanted to confirm the search and accidentally invited the wrong player. Right now nothing happens when you press enter. So what did you want to confirm? Also shouldn't it invite the right player, because you searched for him?
You are right that it is not time critical, but evertime you want to play tmm with your friends you have to invite them, so I wanted to make the invitation process with as few clicks as possible, so you don't get annoyed when you do it for the 50th time.

Reply:
My perspective on this is that the enter to invite seems a little unintuitive to me as well. And accidentally pressing enter while partially searched and inviting someone else seems easy. I think that the loss of pressing enter is not much since the list first shows your friends and then you can just click the button to invite them right away. If you want to invite someone faster you can just add them as a friend

Reply:
Right, I forgot about that friend feature. I am fine this then.